include/nccl_ofi_tuner.h

```diff
@@ -44,9 +45,10 @@ double nccl_ofi_tuner_compute_cost(struct nccl_ofi_tuner_model_dims const* dims,
  * in our configure script, but by definining this manually in cflags, you can
  * choose at plugin build-time which interface to implement. */
 #if defined(AWS_OFI_NCCL_MIN_TUNER_COMPAT) || (AWS_OFI_NCCL_MIN_TUNER_COMPAT <= 1)
-extern const ncclTuner_v2_t ncclTunerPlugin_v2;
+NCCL_OFI_EXPORT_SYMBOL extern const ncclTuner_v2_t ncclTunerPlugin_v2;
 #else
-extern const ncclTuner_v1_t ncclTunerPlugin_v1;
+__attribute__((visibility("default")))
```

Comment:
Don't need this line? (since `NCCL_OFI_EXPORT_SYMBOL` does the same thing)

Reply:
thanks